CHANGELOG.md

```diff
@@ -2,20 +2,68 @@
+
+This feature is enabled by default.
+
+This was added in [pull request #2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164).
+
+### Links are easier to read and have a clearer hover state (opt-in)
+
+Links now have underlines that are consistently thinner and a bit further away from the link text.
+
+Links also have a clearer hover state, where the underline gets thicker to make the link stand out to users.
+
+These changes are an opt-in feature because you may:
+
+- need to make changes to your service to introduce these new link styles consistently
+- want to avoid opting in if you have links within a [multiple-column layout](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Multiple-column_Layout) in your CSS - there is [a Chromium bug that affects such links](https://github.com/alphagov/govuk-frontend/issues/2204)
+
+To enable this feature, set `$govuk-new-link-styles` to `true` before you import GOV.UK Frontend in your Sass:
+
+```
+// application.scss
+$govuk-new-link-styles: true;
+@import "govuk-frontend/all";
+```
+
+If you have created your own link styles, you should review them to ensure their use is consistent across your service.
+
+To make it easier to be consistent in your use of link styles, we’ve also introduced new mixins and modifier classes for:
+
+- removing underlines
+- displaying white text on dark backgrounds
+
+These new link styles will be enabled by default in a future major version of GOV.UK Frontend.
+
+This was added in [pull request #2183: Updates to link styles and link hover states](https://github.com/alphagov/govuk-frontend/pull/2183).
+
+### Style links to remove underlines
+
+You can now remove underlines from links by using either the:
+- `govuk-link-style-no-underline` mixin in your Sass, or
+- `govuk-link--no-underline` class in your HTML
+
+An underline still appears when the user hovers their cursor over the link.
 
-You can now add links styled as buttons to cookie banners.
+This was added in [pull request #2183: Updates to link styles and link hover states](https://github.com/alphagov/govuk-frontend/pull/2183).
 
-Use this feature only if you need to. For example, you can set the **Hide** button to be a link styled as a button that reloads the page. You might want to do this if the cookie banner needs to work without JavaScript and the **Hide** button does not need to set a cookie, as the user has already accepted or rejected cookies.
+### Style links on dark backgrounds
 
-This was added in [pull request #2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+You can now style links on dark backgrounds to use white text colour by using either the:
+- `govuk-link-style-inverse` mixin in your Sass, or
+- `govuk-link--inverse` class in your HTML
```

Comment:
Need to link class to Design System guidance when that becomes available.

Reply:
Based on the current headings in https://github.com/alphagov/govuk-design-system/pull/1625, this should in theory be:

https://design-system.service.gov.uk/styles/typography/#links-on-dark-backgrounds